ios/browser/api/brave_shields/adblock_engine.mm

```diff
@@ -55,11 +55,12 @@ - (AdblockEngineMatchResult*)matchesURL:(NSString*)url
   bool did_match_exception = false;
   bool did_match_important = false;
   std::string redirect = "";
+  std::string rewritten_url = "";
   adblock_engine->matches(
       base::SysNSStringToUTF8(url), base::SysNSStringToUTF8(host),
       base::SysNSStringToUTF8(tabHost), isThirdParty,
       base::SysNSStringToUTF8(resourceType), &did_match_rule,
-      &did_match_exception, &did_match_important, &redirect);
+      &did_match_exception, &did_match_important, &redirect, &rewritten_url);
```

Comment:
This new result should be added to the iOS wrapper as well (`AdblockEngineMatchResult`)

Reply:
updated!